annotationPipeline/src/test/java/org/cbioportal/annotation/TestAnnotationPipeline.java

```diff
@@ -35,22 +35,23 @@
  *
  * @author heinsz
  */
-import org.cbioportal.annotation.pipeline.*;
-import org.springframework.test.context.junit4.*;
-import org.springframework.batch.core.*;
-import org.springframework.batch.test.*; 
+import org.cbioportal.annotation.pipeline.BatchConfiguration;
+import org.cbioportal.annotation.pipeline.TestConfiguration;
 import org.junit.*;
 import org.junit.runner.*;
-import org.springframework.boot.test.IntegrationTest;
-import org.springframework.boot.test.SpringApplicationConfiguration;
-import org.springframework.core.io.FileSystemResource;
+import org.springframework.batch.core.*;
+import org.springframework.batch.test.*;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.*;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@IntegrationTest
-@SpringApplicationConfiguration(classes={BatchConfiguration.class, TestConfiguration.class})
-@TestPropertySource("classpath:application-test.properties")
+@SpringBootTest
+@ContextConfiguration(classes = {BatchConfiguration.class, TestConfiguration.class})
+@TestPropertySource("classpath:application.properties")
```

Comment:
this might be giving some error in circleci:

```
[INFO] Running org.cbioportal.annotation.TestAnnotationPipeline
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.041 s <<< FAILURE! - in org.cbioportal.annotation.TestAnnotationPipeline
[ERROR] pipelineTest  Time elapsed: 0.006 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: java.lang.IllegalStateException: Failed to add PropertySource to Environment
Caused by: java.io.FileNotFoundException: class path resource [application.properties] cannot be opened because it does not exist
```

See: https://app.circleci.com/pipelines/github/genome-nexus/genome-nexus-annotation-pipeline/247/workflows/3fc0e264-c4a2-4256-90ff-eb5871662c23/jobs/458